docs/jirashell.rst

```diff
@@ -18,12 +24,17 @@ This is a specialized Python interpreter (built on IPython) that lets you explor
 Python code is acceptable input. The shell builds a ``JIRA`` client object for you (based on the launch parameters) and
 stores it in the ``jira`` object.
 
-Try getting an issue::
+Try getting an issue
+
+.. code-block:: bash
```

Comment:
Code block python

Reply:
For reference

Before:
![image](https://user-images.githubusercontent.com/26027314/186486196-c8f28415-3815-48d1-a700-daa4e788997f.png)

After (looks the same, which is fine I guess):
![image](https://user-images.githubusercontent.com/26027314/186487544-7c056a71-0987-44ae-a155-36583f41de7e.png)